data/battle_scripts_1.s

```diff
@@ -20,6 +20,34 @@
 
 	.section script_data, "aw", %progbits
 
+BattleScript_Terastallization::
+	@ TODO: no string prints in S/V, but right now this helps with clarity
```

Comment:
Do we want to keep this even if there is no string in SV?

Reply:
My gut says leave it at least for the time being. Since visual indicators are only so good on GBA resolution, I think it helps a lot. I may create an issue for a better Tera animation and that potentially could help.